systemtest/src/main/java/io/strimzi/systemtest/kafkaclients/internalClients/VerifiableClient.java

```diff
@@ -84,6 +182,18 @@ public void setArguments(ClientArgumentMap args) {
         }
     }
 
+    private boolean allowParameter(String minimalVersion, String clientVersion) {
```

Comment:
Do we need this now?

Reply:
spot-bugs already did this spot :D but nice catch 👍